src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5157,6 +5255,283 @@ namespace AirflowNetworkBalanceManager {
 		return CalcWindPressure;
 	}
 
+	Real64
+	AirDensity(
+		Real64 T // Temperature in Celsius
+	)
+	{
+		// Dry air density {kg/m3}
+		// Correlated over the range -20C to 70C
+		// Reference Cengel & Ghajar, Heat and Mass Transfer. 5th ed.
+
+		Real64 const LowerLimit = -20;
+		Real64 const UpperLimit = 70;
+
+		Real64 const a = 1.292;
+		Real64 const b = -0.00467558800659982;
+		Real64 const c = 0.0000125483407801669;
+
+		if ( T < LowerLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for density calculation");
+			return a + b * LowerLimit + c * pow_2( LowerLimit );
```

Comment:
This would be better as:

    if ( T < LowerLimit ) {
        ShowWarningMessage( ... );
        T = LowerLimit;
    } else if ( T > UpperLimit ) {
        ShowWarningMessage( ... );
        T = UpperLimit;
    }
    return a + b * T + c * pow_2( T );

This same comment applies to the other correlation functions